cfg/Pid.cfg

```diff
@@ -33,6 +34,9 @@ gen.add("Kp_ang", double_t, 0, "Kp Angular", 1, 0, 10)
 gen.add("Ki_ang", double_t, 0, "Ki Angular", 0, 0, 2)
 gen.add("Kd_ang", double_t, 0, "Kd Angular", 0.3, 0, 10)
 
+gen.add("lowpass_cutoff", double_t, 0, "Lowpass cutoff (Hz)", 20, 0, 1000)
+gen.add("lowpass_damping", double_t, 0, "Lowpass damping", sqrt(2), 0, 10)
```

Comment:
What default to we want for `lowpass_cutoff`. I could make 0Hz return a special case where the lowpass is disabled for example

Reply:
That's not a bad idea actually. And would also make this PR backward compatible for all controllers not using the D action.

Reply:
I think Raymon meant the lowpass, for which a 10Hz value sounds good, currently we were using half of the sampling frequency and controllers run by defaults at 20Hz if I am not mistaken. 

We also need only to be careful about what happens when the filter frequency is greater than half the sample frequency? we might through and error or an warning right?

Regarding the damping, I would use 0 as default. In most cases only P action suffices.

Reply:
Yes that was my point. If we default to 20Hz while the controller runs on 10 Hz, you the filter will has a worse response that if its off.

The lowpass does still work, but it amplifies certain frequencies a bit.

A lowpass damping of `sqrt(2)` should be the default, because that's critically damped. This is a different parameter that the D action.

Reply:
Ahh yes I was confused with the d action of the PID

Reply:
If the controller runs at 10hz then we should not set it above 5hz

Reply:
So we should not allow this. In dynamic-reconfigure you can deny (or adjust) a change. This would hold for the controller requency as well as the damping.

Reply:
Right now I've made the default that the lowpass is disabled. If you enable the lowpass, the user can choose a good crossover frequency

Reply:
Haha, I should have known that...

Reply:
Yes sorry I didn't mention that. The default lowpass crossover frequency is now 0Hz, which means OFF.